packages/adapter-squid/src/validation/AdapterConfigValidator.ts

```diff
@@ -0,0 +1,27 @@
+import {
+   ERR_INVALID_ADAPTER_CONFIG,
+   IValidator,
+   ValidationError,
+} from "@mate/sdk";
+import { injectable } from "tsyringe";
+import { SquidAdapterConfig, SquidAdapterConfigCodec } from "../types";
+import { isRight, isLeft } from "fp-ts/lib/Either";
+import { PathReporter } from "io-ts/lib/PathReporter";
+
+@injectable()
+export class AdapterConfigValidator implements IValidator<SquidAdapterConfig> {
+   public async validate(data: any): Promise<SquidAdapterConfig> {
```

Comment:
IIRC it's preferable to use `unknown` over `any` to get slightly stricter type checks

```suggestion
   public async validate(data: unknown): Promise<SquidAdapterConfig> {
```